mypyc/lib-rt/CPy.h

```diff
@@ -458,7 +458,7 @@ static inline bool CPyFloat_Check(PyObject *o) {
 
 PyObject *CPy_GetCoro(PyObject *obj);
 PyObject *CPyIter_Send(PyObject *iter, PyObject *val);
-int CPy_YieldFromErrorHandle(PyObject *iter, PyObject **outp);
+int CPy_YieldFromErrorHandle(PyObject *iter, PyObject *outp);
```

Comment:
`PyObject *` can be confusing here. What about using `void *`, and adding a comment saying that it's actually `PyObject **`, but we can't represent it in IR? Alternatively, we could perhaps use `CPyPtr`.

Or maybe the best option would be to add a new primitive type for `PyObject **` (see below for more about this).